jvm-agent/gradle.properties

```diff
@@ -9,4 +9,4 @@
 #
 
 jvmAgentArtifactId=lincheck-jvm-agent
-jvmAgentVersion=0.1-SNAPSHOT
\ No newline at end of file
+jvmAgentVersion=3.1-SNAPSHOT
```

Comment:
Is it expected that `:jvm-agent` version jumped to `3.1`, while `:common` is only at `3.0.2`